api/logic/net/URLConstants.h

```diff
@@ -25,15 +25,15 @@ const QString LIBRARY_BASE("libraries.minecraft.net/");
 //const QString SKINS_BASE("skins.minecraft.net/MinecraftSkins/");
 const QString SKINS_BASE("crafatar.com/skins/");
 const QString AUTH_BASE("authserver.mojang.com/");
-const QString FORGE_LEGACY_URL("http://files.minecraftforge.net/minecraftforge/json");
-const QString FORGE_GRADLE_URL("http://files.minecraftforge.net/maven/net/minecraftforge/forge/json");
-const QString MOJANG_STATUS_URL("http://status.mojang.com/check");
-const QString MOJANG_STATUS_NEWS_URL("http://status.mojang.com/news");
-const QString LITELOADER_URL("http://dl.liteloader.com/versions/versions.json");
+const QString FORGE_LEGACY_URL("https://files.minecraftforge.net/minecraftforge/json");
+const QString FORGE_GRADLE_URL("https://files.minecraftforge.net/maven/net/minecraftforge/forge/json");
+const QString MOJANG_STATUS_URL("https://status.mojang.com/check");
+const QString MOJANG_STATUS_NEWS_URL("https://status.mojang.com/news");
+const QString LITELOADER_URL("https://dl.liteloader.com/versions/versions.json");
```

Comment:
This URL seems to not work with https.

Reply:
And sometimes the URLs are obsolete and no longer used... Maybe they should be removed instead, if there's no remaining use.

Reply:
> This URL seems to not work with https.

Ah yes, checked it served a valid cert but didn't see the 404.

Reply:
> And sometimes the URLs are obsolete and no longer used... Maybe they should be removed instead, if there's no remaining use.

Unused URLs from what I can see:
RESOURCE_BASE
FORGE_LEGACY_URL
FORGE_GRADLE_URL
MOJANG_STATUS_NEWS_URL
LITELOADER_URL